frameworks/hdfs/universe/config.json

```diff
@@ -424,34 +423,34 @@
           "default": 0.25
```

Comment:
Any reason that this one was skipped? Or did we just replace `"integer"`?

Reply:
Just oversight, should not have been skipped.